### ISSUE TITLE
模型中没有Logistic操作还报错不支持Logistic

### ISSUE THREAD

**Issue by changwenkai101** [kendryte/nncase#33]
0.1版本能正常转换的tflite模型，在0.2中报错不支持tflite中的Logistic操作，请问在nncase中什么操作被认为是Logistic？

**Comment by sunnycase** [comment]
1. TFLite 中的 Logistic 对应 tf.nn.sigmoid
2. Logistic 在 nightly-build 中被支持 (https://ci.appveyor.com/project/sunnycase/nncase/branch/master)
